prow-jobs/pingcap-tiproxy-latest-postsubmits.yaml

```
@@ -9,7 +9,7 @@ postsubmits:
       max_concurrency: 1
       skip_report: false
       branches:
-        - ^main$
+        - *
```

```suggestion
        - ".+"
```

please update for all parts.